pbs_scripts/sal_out_collect.script

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -l
+
+################################################################################
+#Script Name: sal_out_collect.pbs
+#Desription : collect salmon output into matrices
+#Args       : dir, genome, outfile suffix
+#Author     : Georgia Doing
+#Date       : 12-18-20
+#Email      : Georgia.Doing.GR@Dartmouth.edu
+################################################################################
+
+##PBS -q default
+#SBATCH --account NCCC
+#SBATCH --job-name=sal_out_col
+#SBATCH --time=02:00:00
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=2
+##PBS -m bea
+#SBATCH --mail-user=Georgia.Doing.GR@Dartmouth.edu
+##PBS -j oe
+##PBS -e oe
+##PBS -o oe
+
+
+
+cd $SLURM_SUBMIT_DIR
```

Comment:
Maybe add a comment here that you ran this on the Dartmouth Discovery cluster so if someone wanted to "in theory" re-run this, they would need to update the location.